internal/server/config/root.go

```diff
@@ -0,0 +1,8 @@
+package config
+
+type Config struct {
+	// Host is the hostname to listen on.
+	Host string `json:"host"`
+	// Port is the port to listen on.
+	Port string `json:"port"`
```

Comment:
Comments not needed in this case.
```suggestion
	Host string `json:"host"`
	Port string `json:"port"`
```